scrapper.py

```diff
@@ -44,18 +77,63 @@ def get_search_urls(self):
         pass
 
 
+class ArticleParser:
+    def __init__(self, full_url, i):
+        self.article_url = full_url
+        self.article_id = i
+        self.article = article.Article
+
+    def _fill_article_with_text(self, article_bs):
+        pass
+
+    """def parse(self):
+        web_page = requests.get(self.article_url, headers=)"""
+
+
+class HTMLParser:
+    pass
+
+
 def prepare_environment(base_path):
     """
     Creates ASSETS_PATH folder if not created and removes existing folder
     """
-    pass
+    try:
```

Comment:
this will work
however, is there a way to implement this without try/except?

Reply:
Reworked this function using shutil